cassiopeia/simulator/TumorWithAFitSubclone.py

```diff
@@ -0,0 +1,141 @@
+"""
+This file stores a subclass of TreeSolver, the TumorWithAFitSubclone. The
+TumorWithAFitSubclone simulates a tumor phylogeny which develops one fit
+subclone.
+"""
+import networkx as nx
+from queue import Queue
+
+from cassiopeia.data import CassiopeiaTree
+from .TreeSimulator import TreeSimulator
+
+
+class TumorWithAFitSubclone(TreeSimulator):
+    r"""
+    Simulates a tumor phylogeny which develops one fit subclone.
+
+    This TreeSimulator simulates a (binary) tumor phylogeny that evolves
+    neutrally with constant branch length 'branch_length_neutral', until in
```

Comment:
I think we should remove explicit references to tumors, since this is useful in other contexts